RevenueCatUI/CustomerCenter/Views/FallbackNoSubscriptionsView.swift

```diff
@@ -82,6 +85,22 @@ struct NoSubscriptionsView: View {
         }
     }
 
+    private var restorePurchasesView: some View {
```

Comment:
Nit: why not "button"?
```suggestion
    private var restorePurchasesButton: some View {
```

Reply:
Works too! I'll go for it